lib/locales/de.js

```diff
@@ -11,7 +11,68 @@
 module.exports = {
     moduleType: 'locale',
     name: 'de',
-    dictionary: {},
+    dictionary: {
+        'Autoscale': 'Automatische Skalierung',                                              // components/modebar/buttons.js:139
+        'Box Select': 'Box Auswahl',                                            // components/modebar/buttons.js:103
+        'Click to enter Colorscale title': 'Klicken, um den Farbskalatitel einzugeben',                       // plots/plots.js:437
+        'Click to enter Component A title': 'Klicken, um den Titel der Komponente A einzugeben',                      // plots/ternary/ternary.js:386
+        'Click to enter Component B title': 'Klicken, um den Titel der Komponente B einzugeben',                      // plots/ternary/ternary.js:400
+        'Click to enter Component C title': 'Klicken, um den Titel der Komponente C einzugeben',                      // plots/ternary/ternary.js:411
+        'Click to enter Plot title': 'Klicken, um den Titel des Graphen einzugeben',                             // plot_api/plot_api.js:579
+        'Click to enter X axis title': 'Klicken, um den Titel der X-Achse einzugeben',                           // plots/plots.js:435
+        'Click to enter Y axis title': 'Klicken, um den Titel der Y-Achse einzugeben',                           // plots/plots.js:436
+        'Compare data on hover': 'Über die Daten fahren, um sie zu vergleichen',                                 // components/modebar/buttons.js:167
+        'Double-click on legend to isolate one trace': 'Daten isolieren durch Doppelklick in der Legende',           // components/legend/handle_click.js:90
+        'Double-click to zoom back out': 'Herauszoomen durch Doppelklick',                         // plots/cartesian/dragbox.js:299
+        'Download plot as a png': 'Download als png',                                // components/modebar/buttons.js:52
+        'Edit in Chart Studio': 'Im Chart Studio bearbeiten',                                  // components/modebar/buttons.js:76
+        'IE only supports svg.  Changing format to svg.': 'IE unterstützt nur SVG-Dateien.  Format wird zu SVG gewechselt.',        // components/modebar/buttons.js:60
```

Comment:
'IE unterstützt nur svg-Dateien.  Format wird zu svg gewechselt.'

Reply:
Wikipedia uses SVG instead of svg. The double space is also in english.